### DISCUSSION TITLE
Logging into a locked vault w/ only 2FA

### DISCUSSION THREAD

**Discussion by bwest2102** [dani-garcia/vaultwarden#4795] [Ideas]
There is a feature I really would like to see in Vaultwarden. When locking the vault, it would be nice to add an option to use your 2nd factor authentication only to unlock the vault instead of the master password or both if needed, for example I have a YubiKey, and it would be nice to unlock it with my YubiKey or my phones passkey function instead of my master password for extra protection. After being the victim of an info stealer malware, I have been committed to make my password manager basically inaccessible without physical access to my YubiKey and computer/homelab server.

**Comment by BlackDex** [comment]
Features like this need to be added to the clients. Which is not something this project maintains. Features like this need to be requested at Bitwarden.

**Reply by bwest2102** [reply]
I contacted Bitwarden and the person I spoke with said that it's specifically Vaultwarden's developers I should be reaching out to in regard to the feature suggestions and support. But I posted here and also on Bitwarden's page as well. I hope to see the feature added in the near future.

**Reply by BlackDex** [reply]
I understand that they say that. Especially if you mention Vaultwarden. But since this project does not maintain the clients or add features to any client that will not happen. 

We only make some small adjustments to get the web-vault working correctly and replace all the brand names. But that is it. 

Also we try to stay as close as possible to Bitwarden in regards of features.
